addons/udes_stock_refactoring/models/stock_move.py

```diff
@@ -143,6 +141,27 @@ def _action_refactor(self, stage=None, refactor_action=False):
                     stage_moves = stage_moves.with_context(remove_related_moves=True)
                     new_moves = refactor_class.do_refactor(stage_moves)
 
+                    moves = (stage_moves | new_moves)
```

Comment:
It will be better to log them separated: like original_moves = XXX, new_moves = XXX